assets/helpers/contributions.js

```diff
@@ -526,14 +440,10 @@ export {
   circlesParse,
   billingPeriodFromContrib,
   errorMessage,
-  getContribKey,
```

Comment:
I thought this list would become much shorter, are we using all the remaining functions somewhere ins the codebase?

Reply:
Yes, combination of the new contributions component and the checkout pages.

Reply:
Ok, If we are looking into rebuilding the checkout page, we should probably need to check whether is possible to simplify the number of functions here. For example, I thought that `circlesParse` would become `parse` now that everything is circles.

Reply:
Yup, that's something that will be possible if we do a rebuild of the checkout pages.